style.css

```diff
@@ -0,0 +1,31 @@
+/* make the navbar fixed to the top of the viewport */
+nav#navbar {
+    position: fixed;
+    height: 5vh;
+    top: 0;
+    width: 100%;
+    font-size: 35px;
+    background-color: rgb(32, 253, 3);
+  }
+  
+  /* make the welcome section the same height as the viewport */
+  section#welcome-section {
+    height: 60vh;
+    margin-left: 37%;
+    margin-top: 40%;
+    font-style: oblique;
+    font-size: 30px;
+  }
+  
+  /* add media queries to adjust the styles for different screen sizes */
+  @media (max-width: 600px) {
+    /* adjust the styles for small screens */
+  }
+  
+  @media (min-width: 601px) and (max-width: 900px) {
+    /* adjust the styles for medium screens */
+  }
+  
+  @media (min-width: 901px) {
```

Comment:
I'd suggest you avoid leaving white spaces before classes. This will make your code cleaner